modules/rubiconBidAdapter.js

```diff
@@ -422,15 +422,16 @@ RubiconAdapter.masSizeOrdering = function(sizes) {
 /**
  * syncEmily
  * @summary A user sync dependency for the Rubicon Project adapter
- * When enabled, creates an user sync iframe after a delay once the first auction is complete.
- * Only fires once except that with each winning creative there will be additional, similar calls to the same service.
- * @example
- *  // Config example for Rubicon user sync
- *  $$PREBID_GLOBAL$$.setConfig({ rubicon: {
- *    userSync: {
- *      enabled: true,
- *      delay: 1000
- *    }
+ * Registers an Emily iframe user sync to be called/created later by Prebid
+ * Only registers once except that with each winning creative there will be additional, similar calls to the same service.  Must enable iframe syncs which are off by default
+@example
+ *  // Config example for iframe user sync
+ *  $$PREBID_GLOBAL$$.setConfig({ userSync: {
+ *    syncEnabled: true,
+ *    pixelEnabled: true,
+ *    syncsPerBidder: 5,
+ *    syncDelay: 3000,
+ *    iframeEnabled: true
```

Comment:
So by turning on the Rubicon adapter, you automatically enable iframe user sync?